src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -319,48 +386,65 @@ public float DualUpdate(float output, float label, float alpha, float invariant,
             return maxNumThreads >= 2 && Math.Abs(fullUpdate) > Threshold ? fullUpdate / maxNumThreads : fullUpdate;
         }
 
-        public Double DualLoss(float label, float dual)
+        public double DualLoss(float label, float dual)
         {
             if (label <= 0)
                 dual = -dual;
 
             // The dual variable is out of the feasible region [0, 1].
             if (dual < 0 || dual > 1)
-                return Double.NegativeInfinity;
+                return double.NegativeInfinity;
 
             return dual * (1 - dual * _halfSmoothConst);
         }
     }
 
     /// <summary>
-    /// Exponential Loss
+    /// Exponential Loss, commonly used in classification tasks.
     /// </summary>
-    public sealed class ExpLoss : IClassificationLoss
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// The Exponential Loss function is defined as:
+    ///
+    /// $L(f(\vec{x}), y) = e^{-\beta y f(\vec{x})}$
+    ///
+    /// where $f(\vec{x})$ is the predicted score, $y \in {-1, 1}$ is the true label, and $\beta$ is a scale factor set to 1 by default.
+    ///
+    /// Note that the labels used in this calculation are -1 and 1, unlike Log Loss, where the labels used are 0 and 1.
+    /// Also unlike Log Loss, $f(\vec{x})$ is the raw predicted score, not the predicted probability (which is calculated by applying a [sigmoid function](https://en.wikipedia.org/wiki/Sigmoid_function) to the predicted score).
+    ///
+    /// The Exponential Loss function penalizes incorrect predictions more than the Hinge loss and has a larger gradient.
+    ///
+    /// ]]>
+    /// </format>
+    /// </remarks>
+    public sealed class ExponentialLoss : IClassificationLoss
```

Comment:
>ExponentialLoss [](start = 24, length = 15)

i prefer code-changes being done in separate PRs. But I'm ok if Shauheen approves this change.

Reply:
We should not change class names at this point.

---
In reply to: [278200927](https://github.com/dotnet/machinelearning/pull/3560#discussion_r278200927) [](ancestors = 278200927)

Reply:
We have been changing all abbreviated forms to the full names. I think the loss functions were overlooked because we were focusing on trainers and transforms. If there is a strong opinion that names should not change now, I will revert this change.

@shauheen @TomFinley @yaeldekel @glebuk

Reply:
Reverting